braindecode/models/usleep.py

```diff
@@ -178,7 +175,6 @@ def __init__(
             input_window_seconds=30,
             time_conv_size_s=9 / 128,
             ensure_odd_conv_size=False,
-            apply_softmax=False,
```

Comment:
we should add `add_log_softmax` param here, in the docstring and use it in `super().__init__`

Reply:
Fixed, thanks!